core/state/statedb.go

```diff
@@ -155,7 +155,7 @@ func New(root common.Hash, db Database, snaps *snapshot.Tree) (*StateDB, error)
 		hasher:              crypto.NewKeccakState(),
 	}
 	if tr.IsVerkle() {
-		sdb.witness = NewAccessWitness(sdb)
+		sdb.witness = sdb.NewFreshAccessWitness()
```

Comment:
Maybe the API name can be improved. '

If someone needs a new access witness to track stuff; should call this API (see L198 below for definition) and get what it wants.